config/default.yml

```diff
@@ -6,6 +6,7 @@ AllCops:
     - "(?:^|/)spec/"
   RSpec/FactoryGirl:
     Patterns:
+    - _spec.rb
```

Comment:
What does this do?

Reply:
The first FactoryGirl cop checked just the factories, so it ignored the actual spec files. I need it to test them.

Reply:
Actually seems unneeded